integration/test/app/kubeconfig/kubeconfig_test.go

```diff
@@ -60,12 +61,27 @@ func TestAppLifecycleUsingKubeconfig(t *testing.T) {
 	{
 		config.Logger.LogCtx(ctx, "level", "debug", "message", "creating kubeconfig secret")
 
-		restConfig := config.Host.RestConfig()
-		bytes, err := kubeconfig.NewKubeConfigForRESTConfig(ctx, restConfig, "test-cluster", targetNamespace)
-		if err != nil {
-			t.Fatalf("expected nil got %#v", err)
+		// Transform kubeconfig file to restconfig and flatten
```

Comment:
Good catch on finding the problem with minikube.

Makes sense to do this now. It gets simpler with the migration to KIND.